build/Scripts/run-integration-tests.ps1

```diff
@@ -20,7 +22,7 @@ if ($saveWorkingFolders) {
 
 $rootDirectory = Resolve-Path "$(Split-Path -Parent $PSCommandPath)\..\.."
 $xUnitPath = Resolve-Path "$rootDirectory\build\Tools\XUnit-Console\xunit.console.exe"
-$unboundedServicesControlPath = Resolve-Path "$rootDirectory\tests\Agent\IntegrationTests\UnboundedServices\unbounded-services-control.ps1"
+$unboundedServicesControlPath = Resolve-Path "$rootDirectory\tests\Agent\IntegrationTests\UnboundedServices\$platform\unbounded-services-control.ps1"
```

Comment:
Rather than move this script into the `linux` subfolder, and presumably duplicate it in the `windows` subfolder, could you leave the script where it is and modify it to handle both cases?

Reply:
That's a good idea.